tests/ui/attributes/unix_sigpipe/unix_sigpipe-and-child-processes.rs

```diff
@@ -0,0 +1,56 @@
+//@ revisions: default sig_dfl sig_ign inherit
+//@ ignore-cross-compile because we run the compiled code
```

Comment:
This isn't present in most run-pass tests, so what makes this one special?

Reply:
Turns out it is not `run-pass` that is the problem here, because the following `run-pass` and `aux-build` test passes in cross-compilation context:

```console
src/ci/docker/run.sh --dev armhf-gnu
../x test --target arm-unknown-linux-gnueabihf tests/ui/cross-crate/reexported-static-methods-cross-crate.rs
```

Instead, it is `aux-bin` that does not support cross compilation yet. For example, `remote-test-client` does not yet know how to upload auxiliary binaries to the target device. That it something that would be nice to solve, but it does not block this PR. I updated the motivation for the ignore.

I would be happy to `@bors r=Mark-Simulacrum` but you'll need to `@bors delegate=Enselic` me first.

Reply:
Sounds good, thanks.